api/src/test/java/bio/terra/tanagra/service/query/api/ApiConversionServiceTest.java

```diff
@@ -147,7 +147,7 @@ void entityDatasetWithOrderBy() {
                     Expression.Literal.create(DataType.INT64, "42")))
             .build(),
         apiConversionService.convertEntityDataset(
-            NAUTICAL_UNDERLAY_NAME, "sailors", apiEntityDataset));
+            NAUTICAL_UNDERLAY_NAME, "sailors", apiEntityDataset, null));
```

Comment:
nit: instead of passing `null` to all places where we're not currently supplying a limit, overloading `convertEntityDataset` to not take that argument and just use a default would make this a little more readable.